3_javascript/javascript2/week3/scripts.js

```diff
@@ -0,0 +1,81 @@
+domListener();
```

Comment:
In general: always test the code that you write. The id name errors would be immediately discovered by running the code.

Reply:
Hi @szoboszlaianna i have made those changes u asked.
please let me know, if i made it right :). thanks for your time..